packages/honeycomb-opentelemetry-web/rollup.config.mjs

```diff
@@ -58,6 +56,46 @@ const esmConfig = {
   plugins: [...modulePlugins],
 };
 
+const IGNORE_WARNINGS = ['THIS_IS_UNDEFINED', 'CIRCULAR_DEPENDENCY', 'EVAL'];
+const printHeader = () => ({
+  name: 'rollup-plugin-print-header',
+  load(source) {
+    if (this.getModuleInfo(source).isEntry) {
+      console.log('⚠️ ignoring warnings: ', IGNORE_WARNINGS.join(', '));
+    }
+  },
+});
+const cdnConfig = {
+  onwarn(warning, defaultHandler) {
+    if (IGNORE_WARNINGS.includes(warning.code)) {
+      return;
+    }
+    defaultHandler(warning);
+  },
+  input: './src/cdn.ts',
+  output: {
+    file: 'dist/umd/index.js',
```

Comment:
Using unpkg it should be available at `https://unpkg.com/@honeycombio/opentelemetry-web/dist/umd/index.js`